keyboards/kbdfans/kbdpad/mk1/rules.mk

```diff
@@ -28,14 +28,16 @@ BOOTLOADER = bootloadHID
 
 # build options
 BOOTMAGIC_ENABLE = no
-MOUSEKEY_ENABLE = yes
+MOUSEKEY_ENABLE = no
 EXTRAKEY_ENABLE = yes
 CONSOLE_ENABLE = yes
-COMMAND_ENABLE = yes
+COMMAND_ENABLE = no
 BACKLIGHT_ENABLE = yes
-RGBLIGHT_ENABLE = no
-RGBLIGHT_CUSTOM_DRIVER = no
+RGBLIGHT_ENABLE = no  # disabled since stock cases don't let underglow show
+RGBLIGHT_CUSTOM_DRIVER = yes
```

Comment:
This was set to "no" because there is no other case that will take this PCB other than the actual kbdfans case. I don't even think you could even get this PCB anymore.

Reply:
PCB is still for sale: https://kbdfans.com/products/kbdfans-kbdpad-pcb-v2. You can also buy new KBDPad cases (I just got one): https://kbdfans.com/products/kbdfans-pad-custom-mechanical-keyboard-diy-kit. I'm not sure if they're clearing stock in favor of KBDPad MKII going forward or not. (They seemed to imply on FB messenger that they intend to stock the MKII at some point in the future after the group buy ships.)

In any case, I agree it makes sense to leave RGBLIGHT_ENABLE off by default. But I think it's still good to set RGBLIGHT_CUSTOM_DRIVER to true by default. That way, if someone does want underglow (they're using a custom case, say), they only have one flag to flip. I don't think it costs extra bytes to have RGBLIGHT_CUSTOM_DRIVER true if RGBLIGHT_ENABLE is false. WDYT?

Reply:
Same reasoning for enabling RGBLIGHT_ANIMATIONS by default; otherwise, code doesn't actually compile when you turn on RGBLIGHT_ENABLE since rgblight_task is not defined.

Reply:
That's a tricky one, do you provide feature support for a PCB even though the keyboard it goes in doesn't support said feature? What if someone makes a custom case is a good argument, however the likelihood of that actually happening is quite small. 

I'm fine with leaving it in since you already did the work.

Reply:
We've had this crop up before. If it's not enabled for the default keyboard it should go in your keymap/rules.mk

Reply:
Sincere question: I understand that for feature enable flags, but what about for config options that are only needed if the feature is enabled, but where there's only one reasonable value if so?

To clarify, this PCB comes with RGB underglow. There are six RGB lights and a custom `rgblight_set` implementation is required, hence my enabling `RGBLIGHT_CUSTOM_DRIVER` and setting `RGBLED_NUM`. Likewise, the standard ps2avrGB RGB code won't actually compile without `RGBLIGHT_ANIMATIONS` defined. So I figured it was a little cleaner to have these options set so that if someone does decide they want to enable underglow, they only have to set `RGBLIGHT_ENABLE` and things will just work.

That said, the cases KBDfans sold for this don't really let the underglow be visible, and I agree it's fairly unlikely someone will use this PCB in a custom case. Thus I don't have strong feelings about this. :) So I do not mind reverting the RGB changes, no big loss there. (I am curious about the general pattern when a PCB has more features than the stock case, though, in case it's relevant to future PRs of mine.)

Reply:
So if we do want to keep the RGB bits, it might make sense to wait till after #7183 lands so we can strip away the custom `rgblight_set` entirely. WDYT?

Reply:
Replace RGB code with standard ps2avrGB driver.